src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,29 +6,49 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact {
+
+    public static final String EXAMPLE = "a/123, Clementi Ave 3, #12-34, 231534";
+    // public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.